package/MDAnalysis/coordinates/PDB.py

```diff
@@ -391,14 +391,25 @@ def _read_frame(self, frame):
             elif line[:6] == 'CRYST1':
                 # does an implicit str -> float conversion
                 try:
-                    self.ts._unitcell[:] = [line[6:15], line[15:24],
-                                            line[24:33], line[33:40],
-                                            line[40:47], line[47:54]]
+                    cell_dims = np.array([line[6:15], line[15:24],
+                                  line[24:33], line[33:40],
```

Comment:
I'm not sure if it's a PEP8 or just a flake8 thing, but my editor is giving me some warning about this indentation. Can you shift the start of these lines so that the first character lines up with the first "[" from the line above?